src/main/java/com/cloudogu/smeagol/wiki/domain/WikiId.java

```diff
@@ -0,0 +1,45 @@
+package com.cloudogu.smeagol.wiki.domain;
+
+import java.util.Objects;
+
+/**
+ * Wiki identifier, a wiki can be identified by the id of the repository and its branch.
+ */
+public class WikiId {
+
+    // TODO should be name and repositorID ???
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS1135)